dev-resources/puppetlabs/services/ca/certificate_authority_core_test/master/conf/ssl/ca/infra-crl.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN X509 CRL-----
```

Comment:
What does this do and how does it interact with the tests?

Reply:
It is dummy (copy of existing crl.pem in same folder). I stumbled upon the need for the file to exist when testing the certificate_authority_core_test.clj (crl-endpoint-test) when passing infra-crl-path to handle-get-certificate-revocation-list. crl-endpoint-test by itself does not use/need it. Since the unit test do not run the CA service that file is not created as it happens when running integration tests.